code/leslie/Python/lab07.py

```diff
@@ -0,0 +1,100 @@
+import math # because I'm using math.ceil() to round up the answer
```

Comment:
[Lab 7: Compute Automated Readability Index](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/07%20ARI.md)

**Growth Opportunity**

* Need to use 'relative' path for text file instead of 'absolute' path. The filepath me be determined by what directory we run the function from or where the python file is located. We can discuss if needed.
    * Try something like the following:
        * '\\gettysburg_address.txt'
    * Instead of:
        * 'code\\leslie\\Python\\gettysburg_address.txt'